sources/scraping/scrapy/spider.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any, Optional, TypeVar
+import scrapy  # type: ignore[import-untyped]
+
+from ..queue import BaseQueue
+
+T = TypeVar("T")
+
+
+class DltSpider(scrapy.Spider):
```

Comment:
why would our Spider need a queue? We do not need `DltSpider` now

Reply:
The reason for this is that we need to pass queue instance to Item Pipeline and at the moment there is no officially documented way to pass queue instance in Item pipeline we have defined

Reply:
Other option would be defining ItemPipeline inside scrapy runner and passing to crawler process something like
```py
lass ScrapyRunner(Runnable):
    def run(self, *args: Any, **kwargs: AnyDict) -> None:
        queue = self.queue
        class ItemPipeline:
              def process_item(...):
                   queue.put(...)
       
        runner = CrawlerProcess(settings={
              "ITEM_PIPELINES": {
                  ItemPipeline: 100
              }
        })
```

Reply:
We no longer need out custom spider.